packages/lerna-semantic-release-pre/index.js

```diff
@@ -89,26 +99,87 @@ function bumpVersionCommitAndTag (nextRelease, done) {
   });
 }
 
+function updateDependencies (pkg, updates, dependenciesKey) {
```

Comment:
Maybe I'm not fully understanding how this works, but is the idea to update the *dependents* of a package, rather than the dependencies?